rpc_differ/rpc_differ.py

```diff
@@ -94,6 +95,12 @@ def create_parser():
         default="https://github.com/rcbops/rpc-openstack",
         help="Github repository for the rpc-openstack project"
     )
+    parser.add_argument(
+        '-rp', '--rpc-product',
+        action='store',
+        default="master",
+        help="Set the RPC product version"
+    )
     parser.add_argument(
```

Comment:
so as well as defining the old_commit and new_commit for rpco, we also need to pass the rpc-product in?

Reply:
yes rpco master can deploy any version of our supported product. this was recently completed as part of the modernization effort.